### DISCUSSION TITLE
2022-03-21 Core team update

### DISCUSSION THREAD

**Discussion by mm-near** [near/community#232] [Core]
# TL;DR

* tools that we are building (for localnet loadtesting and mock_network) - are starting to show first results and allow us to find the bottlenecks of our system
* we keep investing in more metrics and debug information to understand better what's happening to the system (this week - we got a new tool for network)

* designing for the storage growth - with cold storage that would allow us to keep old archival data off SSD.

# Done
@Longarithm - chunk nodes cache implementation merged (nearcore/6414)
@Longarithm - but not activated yet, as we have to do more performance analysis
@Longarithm - found inaccuracy in TouchingTrieNodeWrite estimation (nearcore/6464)

@mzhangmzz - found reason why mock_network was slow (after looking at lots of flame graphs)
@mzhangmzz - improving documentation on sync & catchup
@mzhangmzz - adding metrics for measuring time around partial chunk requests

@EdvardD - design for cold storage is almost ready
@EdvardD - experimenting with SST file sizes and file opens - to see if our rocksdb settings can be improved

@pompon0 - tool to discover all the nodes in the network and analyse network topology
@pompon0 - design for the network improvements

@mm-near - PR with more information on the upcoming blocks (nearcore/6466)
@mm-near - resubmitting the client-actor on separate thread (the previous PR was rolled back due to issues in our testing)
@mm-near - investigating the mainnet issues with validators stuck at the beginning of epoch (one theory and additional documentation)

@jc-near - sync debug page submitted
